gossip/src/weighted_shuffle.rs

```diff
@@ -388,14 +388,14 @@ mod tests {
     fn test_weighted_shuffle_zero_weights() {
         let weights = vec![0u64; 5];
         let seed = [37u8; 32];
-        let mut rng = ChaChaRng::from_seed(seed);
+        let mut rng = ChaCha8Rng::from_seed(seed);
```

Comment:
It's a little confusing that our tests have a mix of ChaCha8 and ChaCha20. I think we should test one, the other, or both